spectre-meltdown-checker: add page (#5670)
tldr-pages/tldr: pages/linux/spectre-meltdown-checker.md

# spectre-meltdown-checker

> Spectre and Meltdown mitigation detection tool.
> More information: <https://manned.org/spectre-meltdown-checker.1>.

> - Check the currently running kernel for Spectre and Meltdown:

Reviewer: ```suggestion
- Check the currently running kernel for Spectre or Meltdown:
```
?

> `spectre-meltdown-checker`

Reviewer 2: Without `sudo` I get this warning:
```
Note that you should launch this script with root privileges to get accurate information.
We'll proceed but you might see permission denied errors.
To run it as root, you can try the following command: sudo /usr/bin/spectre-meltdown-checker
```
So, maybe we should add `sudo` to the commands. What do you think @bl-ue ?

Reviewer: Yeah, make sense to me.

Author: I completely overlooked that. It is even mentioned in the Github repository.

> - Check the currently running kernel and show explanations for taken actions to mitigate a vulnerability:

Reviewer: ```suggestion
- Check the currently running kernel and show an explanation of the actions to take in order to mitigate a vulnerability:
```

`spectre-meltdown-checker --explain`

> - Don't use the `/sys` interface even if present:

Reviewer: How about moving this example down below the next example, i.e. to the 4th position?  the next one seems more essential.

Author: Sounds reasonable

`spectre-meltdown-checker --no-sysfs`

> - Check for specific variants (`1`, `2`, `3`, `3a`, `4`, `l1tf`, `msbds`, `mfbds`, `mlpds`, `mdsum`, `taa`, `mcespc`, `srbds` - defaults to all):
> 
> `spectre-meltdown-checker --variant {{3a}} --variant {{mcespc}}`

Reviewer: ```suggestion
- Check for specific variants (defaults to all):

`spectre-meltdown-checker --variant {{1|2|3|3a|4|l1tf|msbds|mfbds|mlpds|mdsum|taa|mcespc|srbds}}`
```

> - Inspect a non-running kernel:

Reviewer: to be more consistent with your previous wording...
```suggestion
- Check a non-running kernel:
```

# spectre-meltdown-checker

> Spectre and Meltdown mitigation detection tool.
> More information: <https://manned.org/spectre-meltdown-checker.1>.

- Check the currently running kernel for Spectre or Meltdown:

`sudo spectre-meltdown-checker`

- Check the currently running kernel and show an explanation of the actions to take in order to mitigate a vulnerability:

`sudo spectre-meltdown-checker --explain`

- Check for specific variants (defaults to all):

`sudo spectre-meltdown-checker --variant {{1|2|3|3a|4|l1tf|msbds|mfbds|mlpds|mdsum|taa|mcespc|srbds}}`

> - Print the output using a specific formatting:

Reviewer 2: ```suggestion
- Display output using a specific output format:
```
